src/Runner.Worker/ExecutionContext.cs

```diff
@@ -272,7 +272,7 @@ public void RegisterPostJobStep(IStep step)
                 return;
             }
 
-            step.ExecutionContext = Root.CreatePostChild(step.DisplayName, IntraActionState);
+            step.ExecutionContext = Root.CreatePostChild(step.DisplayName, IntraActionState, ContextName);
```

Comment:
do we normally set ContextName for pre/post? can pre/post actually contribute to step outputs?